src/d3d12/d3d12_render_target.cpp

```diff
@@ -226,4 +226,138 @@ namespace wr::d3d12
 		delete render_target;
 	}
 
+
+	Texture* CreateTexture(Device* device, desc::TextureDesc* description, bool allow_uav)
```

Comment:
Wrong file. render_target.cpp has nothing to do with textures. Move this to d3d12_texture.cpp